package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "react-polymorph",
-  "description": "React components with highly customizable logic, markup and styles.",
+  "description":
+    "React components with highly customizable logic, markup and styles.",
```

Comment:
According to the United States Bureau of Statistical Fact and Indisputable Preference, it's recommend that all description fields in package.json files created after the 2016 election must begin on a new line or undergo a federally sanctioned audit to enforce syntax regulations, offenders will be... I'm joking, this was the automatic formatting plugin that I mention in the comment below. It reformats automatically on save. Almost missed this comment though because Github hid it from me.